### PR TITLE
OS#18260893 [OSS-Fuzz #9267] [OSS-Fuzz #9326] avoid assertion for re-start/stop in debugger mode

### PR DESCRIPTION
**Pull request by duongnhn**

### CODE REVIEW

File: test/Debugger/JsDebuggerAttachDetach.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+eval("var x = '🇦';");
```
Comment (kfarnung) [inline test/Debugger/JsDebuggerAttachDetach.js]: What's the purpose of this line?

File: test/Debugger/JsDebuggerAttachDetach.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+eval("var x = '🇦';");
+
+function foo() {
+    var x = 1; /**bp:dumpSourceList();**/
```
Comment (kfarnung) [inline test/Debugger/JsDebuggerAttachDetach.js]: We should probably have this print something, I think we expect this to run on both detaches but the test doesn't currently validate that.

**Comment by digitalinfinity** [comment]
@duongnhn was this bug found through oss-fuzz? If so, we should credit them- @akroshg does the credit need to be provided in the commit message?

**Comment by duongnhn** [comment]
yes, will do

---
In reply to: [411627408](https://github.com/Microsoft/ChakraCore/pull/5586#issuecomment-411627408) [](ancestors = 411627408)

**Pull request merged**